pgm-2.txt

```diff
@@ -0,0 +1,7 @@
+
+#!/bin/bash
+echo "hellooooooo"
+find *.log > z.txt
+find *.txt >> z.txt
+sudo mkdir /home/sudhir/zout
+tar -zczf matches.tar.gz /home/main
```

Comment:
The requirement is to tar and compress only the *.txt and *.log from this folder. tar and compress should ignore all other files in this folder. Can you accomplish that?